### PR TITLE
[Chore] Upgrade stack versions

### PR DESCRIPTION
**Pull request by andyduong1920**
## What happened

Upgrade stack versions

## Insight

Automatically created by the Upgrade stacks workflow.

## Proof Of Work

On the Files changed tab

### CODE REVIEW

**Comment by Nihisil** [comment]
Does it not work with Alpine 3.17? What are the reasons for downgrading it?

**Comment by andyduong1920** [comment]
> Does it not work with Alpine 3.17? What are the reasons for downgrading it?

It due to the issue when installing `glibc-2.34-r0.apk`, I'm trying with `glibc-2.35-r0.apk`

**Comment by andyduong1920** [comment]
> Does it not work with Alpine 3.17? What are the reasons for downgrading it?

Tried with glibc-2.35-r0.apk in Alpline 3.17.0 and it also doesn't work FYI @Nihisil 

image

**Comment by Nihisil** [comment]
@andyduong1920 Sorry, I had intended to push these changes to a separate branch, but I accidentally pushed them directly to your branch. Let's see if the CI build passes or not, and after that we can rebase this branch to exclude my commit.

File: lib/nimble_template/projects/project.ex [hunk 1]
```diff
@@ -1,10 +1,10 @@
 defmodule NimbleTemplate.Projects.Project do
   @moduledoc false
 
-  @alpine_version "3.14.6"
-  @elixir_version "1.14.0"
-  @erlang_version "25.0.4"
-  @node_asdf_version "16.15.0"
+  @alpine_version "3.17.2"
```
Comment (andyduong1920) [inline lib/nimble_template/projects/project.ex]: The latest alpine version on hex docker image is 3.17.0 FYI @Nihisil :) 

https://hub.docker.com/layers/hexpm/elixir/1.14.3-erlang-25.2.3-alpine-3.17.0/images/sha256-55a63447e18a47f72b811f89dea73a3b9307e91fa6b01139e0022d0d9ae2a8d9?context=explore

File: priv/templates/nimble_template/Dockerfile.eex [hunk 2]
```diff
@@ -16,7 +16,7 @@ RUN apk update && \
 <%= if web_project? do %>
 RUN wget -q -O /etc/apk/keys/sgerrand.rsa.pub https://alpine-pkgs.sgerrand.com/sgerrand.rsa.pub
 RUN wget https://github.com/sgerrand/alpine-pkg-glibc/releases/download/2.34-r0/glibc-2.34-r0.apk
-RUN apk add glibc-2.34-r0.apk
+RUN apk add --force-overwrite glibc-2.34-r0.apk
```
Comment (andyduong1920) [inline priv/templates/nimble_template/Dockerfile.eex]: Hopefully the --force-overwrite will work, if yes, please also consider to upgrade to `glibc-2.35-r0.apk` too @Nihisil 👍

**Comment by andyduong1920** [comment]
> @andyduong1920 Sorry, I had intended to push these changes to a separate branch, but I accidentally pushed them directly to your branch. Let's see if the CI build passes or not, and after that we can rebase this branch to exclude my commit.

No worry @Nihisil you're free to test on this branch :D

**Review by Nihisil — APPROVED** [review]
It looks like there is no easy solution for alpine upgrade, after `glibc` there is another error with `dart-sass`. I have created ticket to fix these issues later: https://github.com/nimblehq/elixir-templates/issues/319

**Review by olivierobert — APPROVED** [review]

**Review by nvminhtue — APPROVED** [review]

**Pull request merged**